frontend/src/features/splReports/containers/SplReportContainer.tsx

```diff
@@ -33,17 +46,52 @@ const SplReportContainer: React.FunctionComponent<ISplReportContainerProps> = ()
     updateProjectReport,
     exportReport,
   } = useProjectSnapshotApi();
+  const [snapshotFilter, setSnapshotFilter] = React.useState<ISnapshotFilter>({
+    projectNumber: '',
+    agency: '',
+    fiscalYear: '',
+    sortBy: { projectId: 'asc', snapshotOn: 'desc' },
+  });
 
   const id = currentReport?.id;
   useEffect(() => {
     const fetch = async () => {
       if (id) {
-        const data = await getProjectReportSnapshotsById(id);
+        let data = await getProjectReportSnapshotsById(id);
+        data = data.map(
+          s =>
+            ({
+              ...s,
+              project: { ...s.project, agencyName: s.project?.agency || s.project?.subAgency },
+            } as any),
```

Comment:
A lot of `any` in this component.  It would be nice to not lose the reason we're using typescript.